src/SamplesApp/SamplesApp.macOS/SamplesApp.macOS.csproj

```diff
@@ -140,44 +140,34 @@
     <ImageAsset Include="Assets.xcassets\unologo.imageset\Contents.json" />
     <ImageAsset Include="Assets.xcassets\unologo.imageset\unoplatform.jpg" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Resources\" />
-    <Folder Include="Resources\Fonts\" />
-    <Folder Include="Assets.xcassets\unologo.imageset\" />
-  </ItemGroup>
+  <ItemGroup />
   <ItemGroup>
     <None Include="Info.plist" />
     <None Include="Entitlements.plist" />
+    <None Include="Resources\Fonts\uno-fluentui-assets.ttf" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Main.cs" />
     <Compile Include="AppDelegate.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <BundleResource Include="Resources\Fonts\winjs-symbols.ttf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>

```

Comment:
same here

Reply:
actually in this case it's worse, seems like the `BundleResource` build action got removed